pkg/bpf/bpffs.go

```diff
@@ -122,6 +122,29 @@ func isBpffs(path string) bool {
 	return int32(magic) == int32(fsdata.Type)
 }
 
+type buffer struct {
```

Comment:
Can you move this into `pkg/lockbuffer` or similar? I'm pretty sure we'll need this again.